store/mockstore/mocktikv/mvcc_leveldb.go

```diff
@@ -566,8 +566,15 @@ func prewriteMutation(db *leveldb.DB, batch *leveldb.Batch, mutation *kvrpcpb.Mu
 	if err != nil {
 		return errors.Trace(err)
 	}
+
+	// Check assertions.
+	if (ok && mutation.Assertion == kvrpcpb.Assertion_NotExist) ||
+		(!ok && mutation.Assertion == kvrpcpb.Assertion_Exist) {
+		log.Error("ASSERTION FAIL!!!", mutation)
```

Comment:
I think you want to log the assertion type too.

Reply:
The `mutation` contains all the information we need, it has:

```
Op = Put | Del | Lock ...
Assertion = NotExist | Exist ...
```